srv/salt/ceph/stage/discovery/default.sls

```diff
@@ -4,9 +4,17 @@ ready:
     - name: minions.ready
     - timeout: {{ salt['pillar.get']('ready_timeout', 300) }}
 
+refresh_pillar0:
+  salt.state:
+    - tgt: {{ salt['pillar.get']('master_minion') }}
```

Comment:
yea, since at this stage only the global/cluster.yml would be populated, everything else would be during/after stage2